content/blogs/2024-10-15-deploying-kestra-in-clever-cloud.md

```diff
@@ -0,0 +1,177 @@
+  repository:
+    type: postgres
+  # Configure MinIO storage with the env vars from the add-on <3>
+  storage:
+    type: minio
+    minio:
+      endpoint: https://${CELLAR_ADDON_HOST}
+      port: 80
+      accessKey: ${CELLAR_ADDON_KEY_ID}
+      secretKey:  ${CELLAR_ADDON_KEY_SECRET}
+      region: US
+      bucket: kestra-internal-storage
+  queue:
+    type: postgres
+  tasks:
+     tmp-dir:
+      path: /tmp/kestra-wd/tmp
+  # Setup the URL to the CleverCloud host <4>
+  url: ${APP_ID}.cleverapps.io
+  # As the Docker engine is not accessible, configure globally the Process runner for all plugins <5>
+  plugins:
+    defaults:
+      - type: io.kestra.plugin.scripts
+        values:
+          taskRunner:
+            type: io.kestra.plugin.core.runner.Process
+```
+
+1. We configure Kestra to use PostgreSQL as its backend by using the environment variables injected from the add-on (more on this later).
+2. The application will be available publicly, so we set a user and password. I strongly recommend you to do the same.
+3. We configure Kestra to use the MinIO storage by using the environment variables injected from the add-on. The MinIO storage works with all S3 compatible storage including Cellar.
+4. We set the URL of Kestra to the URL of the application, by default, this will be `${APP_ID}.cleverapps.io` where `APP_ID` is an environment variable injected by Clever Cloud with the identifier of your application.
+5. As the Docker engine is not accessible from the Kestra container, we configure globally the `Process` task runner for all plugins via plugins default.
+
+![Step 3 - Docker](/blogs/2024-10-15-deploying-kestra-in-clever-cloud/clever-cloud-step-3-docker.png)
+
+Remember to click on **UPDATE CHANGES** before clicking on **NEXT** to save your changes. You will then arrive at a page which explains how to deploy a Docker application via git push, follow the instructions on this page and push your Git branch to Clever Cloud.
+
+![Step 4 - Docker](/blogs/2024-10-15-deploying-kestra-in-clever-cloud/clever-cloud-step-4-docker.png)
+
+After a few seconds, the console will detect the push and switch to the logs of the deployment, as we're missing the necessary services, you can abort the deployment.
+
+![Step 5 - Docker](/blogs/2024-10-15-deploying-kestra-in-clever-cloud/clever-cloud-step-5-docker.png)
+
+## Creating a Cellar bucket
+
+On the Clever Cloud console, click on **CREATE**, select **an add-on** then **Cellar S3 storage**.
+
+![Step 6 - Cellar](/blogs/2024-10-15-deploying-kestra-in-clever-cloud/clever-cloud-step-6-cellar.png)
+
+There is only one plan available, so click on **NEXT**, then click on **LINK** in front of the `kestra-clever` to link the add-on to the application. Linking the add-on will inject environment variables to the linked application with connection URL and credentials so it can be easily configured without needed to hardcode them. I would say this is a nice feature ;).
```

Comment:
```suggestion
There is only one plan available, so select **NEXT**, then select **LINK** in front of the `kestra-clever` to link the add-on to the application. Linking the add-on will inject environment variables to the linked application with the connection URL and credentials so it can be easily configured without needed to hardcode them. This is a very nice feature 😉.
```